Engine/src/Console/CommandsHistory.bf

```diff
@@ -0,0 +1,75 @@
+using System;
+
+namespace SteelEngine.Console
+{
+	class CommandsHistory
+	{
+		private String[] _history ~  DeleteContainerAndItems!(_);
```

Comment:
Why not just use a `List` since this is expandable?

Reply:
Okay, I see later on that we have a max history length. The array is great then for performance reasons, but we should also still consider allowing the history length to change at runtime and handle the copying of old history items to the new container if need be.